app/views/tasks/index.html.erb

```diff
@@ -0,0 +1,58 @@
+
+
+<section id="index_container">
+  <h2 id="to_do"> # TODO</h2>
+  <ul class="to_do_list">
+    <% counter = 0 %>
+    <% @tasks.each do |task| %>
+    <% counter += 1 %>
+    <li>
+      <%= counter %>)
+      <span <%= task.status ? "class=strikethrough"  : "class=normal" %>>
+
+          <%= link_to(task.name, task_path(task.id), alt: "link to task name", class: "task_link") %>
+      </span>
+
+      <span class="sublist_and_box">
+        <ul class="bullet_list">
+          <li >
+            <form>
+
+            <div class="squaredThree">
+              <label for="checkbox-input" <%= task.status ? "class=strikethrough"  : "class=normal" %>>
+
+                  <%= link_to("Initial Commit", mark_initial_complete_path(task.id), {method: :patch, alt: "link to mark initial complete", class: "task_link"}) %>
+              </label>
+              <input type="checkbox" id="checkbox-input" value="None" name="check" <%= task.initial_status ? "checked" : "unchecked" %> />
+              <label for="squaredThree"></label>
```

Comment:
the property "for" on labels actually matches with an id, and I see that you only have classes labeled as "squaredThree"

Also, this label tag is empty and should probably be removed